backend/python/app/services/implementations/story_service.py

```diff
@@ -120,3 +120,14 @@ def get_story_translations(self, user_id, translator):
         except Exception as error:
             self.logger.error(str(error))
             raise error
+
+    def get_story_available_for_review(self, level, language):
+
+        stories = (
+            Story.query.join(StoryTranslation)
+            .filter(Story.level <= level)
+            .filter(StoryTranslation.language == language)
+            .filter(StoryTranslation.stage == "REVIEW")
```

Comment:
wrong filter: you're looking for something like `StoryTranslation.review_id == none`

BTW: a story translation that is available for review is in `TRANSLATION` stage, not `REVIEW` stage. The story is in `REVIEW` stage when a reviewer is assigned.